clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -332,6 +359,15 @@ class device_ext : public sycl::device {
     int max_nd_range_size[] = {0x7FFFFFFF, 0x7FFFFFFF, 0x7FFFFFFF};
     prop.set_max_nd_range_size(max_nd_range_size);
 
+    // Simulate the max register size per work group = (max work group size /
+    // min sub group size) * (each hardware thread register num *
+    // each register size).
+    // The hardware thread register num 128 and the register size 32 bytes are
+    // based on the Xe architecture graphics EU. Feel free to update the value
```

Comment:
Is it possible to use more generic numbers that would make sense for most of SYCL devices?